src/components/Modal/Modal.test.js

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import { shallow } from 'enzyme';
 import Modal from './index';
 
-describe('<Modal />', () => {
+describe('<Modal/>  component', () => {
```

Comment:
Test cases should read as a sentence please refactor this to `The modal component`

Reply:
I have refactored it to `The modal content` as advised. Thanks